jans-cedarling/cedarling/src/common/policy_store.rs

```diff
@@ -0,0 +1,159 @@
+/*
+ * This software is available under the Apache-2.0 license.
+ * See https://www.apache.org/licenses/LICENSE-2.0.txt for full text.
+ *
+ * Copyright (c) 2024, Gluu, Inc.
+ */
+
+use std::collections::HashMap;
+
+use super::cedar_schema::CedarSchema;
+
+/// PolicyStoreMap it is a store for `PolicyStore` accessible by key.
+#[derive(Debug, serde::Deserialize)]
+pub(crate) struct PolicyStoreMap {
+    #[serde(flatten)]
+    pub policy_stores: HashMap<String, PolicyStore>,
+}
+
+/// PolicyStore contains all the data the Cedarling needs to verify JWT tokens and evaluate policies
+#[derive(Debug, Clone, serde::Deserialize)]
```

Comment:
could you add the other fields from the [docs](https://github.com/JanssenProject/jans/wiki/Cedarling-Nativity-Plan#cedarling-policy-store)?

Reply:
No, it is not scope of the https://github.com/JanssenProject/jans/issues/9854

Reply:
All changes to the documentation should be implemented in the corresponding issue.

Reply:
ok